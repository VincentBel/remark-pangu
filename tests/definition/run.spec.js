runSpec(__dirname)
runSpec(__dirname, {
    definition: false
})