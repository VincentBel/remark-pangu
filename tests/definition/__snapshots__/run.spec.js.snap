// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`definition.md with options: {"definition":false}: definition.md 1`] = `
[链接a引用1][link]

![图片b引用2][link]

[link]: http://example.com/链接a链接1 "标题a标题1"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[链接 a 引用 1][link]

![图片 b 引用 2][link]

[link]: http://example.com/链接a链接1 "标题a标题1"

`;

exports[`definition.md with options: undefined: definition.md 1`] = `
[链接a引用1][link]

![图片b引用2][link]

[link]: http://example.com/链接a链接1 "标题a标题1"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[链接 a 引用 1][link]

![图片 b 引用 2][link]

[link]: http://example.com/链接a链接1 "标题 a 标题 1"

`;
